conformance/tests/httproute-response-header-modifier.go

```diff
@@ -168,6 +168,44 @@ var HTTPRouteResponseHeaderModifier = suite.ConformanceTest{
 			},
 			Backend:   "infra-backend-v1",
 			Namespace: ns,
+		}, {
+			Request: http.Request{
+				Path: "/response-and-request-header-modifiers",
+				Headers: map[string]string{
+					"X-Header-Remove":     "remove-val",
+					"X-Header-Add-Append": "append-val-1",
+				},
+			},
+			BackendSetResponseHeaders: map[string]string{
+				"X-Header-Set-2":    "set-val-2",
+				"X-Header-Add-2":    "add-val-2",
+				"X-Header-Remove-2": "remove-val-2",
+				"Another-Header":    "another-header-val",
+				"X-Header-Remove-1": "remove-val-1",
```

Comment:
Maybe add one here that is not modified in any way to show that it passes all the way through.

Reply:
Thanks for the suggestion Rob! I have added it in.